internal/runtime/compiler/parser/parser.y

```diff
@@ -142,6 +149,27 @@ stmt
   }
   ;
 
+log_mapping_spec
```

Comment:
Then this should just be

```
log_filter_stmt
 : LOG_FILTER log_filter_list
 ;
```

It's weird to me that you need a NL here unlike the other stmt rules.

Reply:
Done